### DISCUSSION TITLE
I cant activate my Github Copilot :(

### DISCUSSION THREAD

**Discussion by lauro-tup-student** [community/community#47951] [GitHub Education]
I wanted to try the GitHub CoPilot but I can seem to activate it. Please see image below.

I am following the tutorial on how to activate it via Settings but the button is different from what I'm seeing

![image](https://user-images.githubusercontent.com/125790056/219937537-05fffcc2-cded-4849-b93c-7d452e86f1cb.png)

**Comment by jge162** [comment]
Have you actually tried installing the extension on an IDE and then authenticate it that way?

**Comment by lauro-tup-student** [comment]
Hi Jermyiah,

Thank you very much for reaching out. Yes, I already installed the
extension on my Pycharm Professional IDE. I hope you can help me. I wanted
to test the the copilot on my current project :(

Thank you,


Regards,
Lau

**Comment by AsafMeizner** [comment]
GitHub copilot isn't free beta anymore, you need to either pay for it or be eligible to get it for free eg:(GitHub student developer pack, teacher, or your school pays for school pack),

hope this helps :)
